README.md

```diff
@@ -12,7 +12,7 @@ If you use [maven](https://maven.apache.org/what-is-maven.html), just include fo
 <dependency>
   <groupId>us.abstracta.jmeter</groupId>
   <artifactId>jmeter-java-dsl</artifactId>
-  <version>0.35</version>
+  <version>0.36</version>
```

Comment:
seems something went wrong on last release or I just stepped into this change inadvertently, sorry for this.